src/Tribe/Admin/Notice/Marketing/Stellar_Sale.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Notice for the Stellar Sale
+ *
+ * @since TBD
+ */
+
+namespace Tribe\Admin\Notice\Marketing;
+
+/**
+ * Class Stellar_Sale
+ *
+ * @since TBD
+ *
+ * @package Tribe\Admin\Notice\Marketing
+ */
+class Stellar_Sale extends \Tribe\Admin\Notice\Date_Based {
+	/**
+	 * {@inheritDoc}
+	 */
+	public $slug = 'stellar-sale';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public $start_date = 'July 28th, 2021';
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * 1pm UTC is 6am PDT (-7) and 9am EDT (-4)
+	 */
+	public $start_time = 13;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public $end_date = 'August 4th, 2021';
```

Comment:
Not worried about future years - this should only run if the date is changed. 
It's year-specific so we don't accidentally run it next year because we forgot to change it.